paparazzi-gradle-plugin/src/test/java/app/cash/paparazzi/gradle/PaparazziPluginTest.kt

```diff
@@ -1505,6 +1505,15 @@ class PaparazziPluginTest {
       .runFixture(fixtureRoot) { build() }
   }
 
+  @Test
+  fun robolectric() {
+    val fixtureRoot = File("src/test/projects/robolectric")
+
+    gradleRunner
+      .withArguments("verifyPaparazziDebug", "--stacktrace")
```

Comment:
```suggestion
      .withArguments("testPaparazziDebug", "--stacktrace")
```

no need to verify, this test only wants to ensure Paparazzi's ASM + Robolectric can co-exist.